### PR TITLE
Added params and auditd rules for puppet-cis module

### PR DESCRIPTION
**Pull request by mendoncaangelo**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -19,6 +19,8 @@
   $sender_hostname = 'sender.example.org'
   $masquerade_domains = 'example.org'
   $relayhost = 'receiver.example.org'
+  $ssh = false
```
Comment (swizzley) [inline manifests/params.pp]: default should be true

File: manifests/params.pp [hunk 2]
```diff
@@ -19,6 +19,8 @@
...
+  $postfix = false
```
Comment (swizzley) [inline manifests/params.pp]: default should be true

**Comment by swizzley** [comment]
will merge once defaults are set to true

File: manifests/params.pp (hunk 1 above)
Comment (mendoncaangelo) [reply manifests/params.pp]: 👍

File: manifests/params.pp (hunk 2 above)
Comment (mendoncaangelo) [reply manifests/params.pp]: 👍

**Comment by mendoncaangelo** [comment]
Hi Dustin, Will you be releasing a new version to puppet forge anytime soon?

**Comment by swizzley** [comment]
No

**Pull request merged**